docs/man/dune.1.md

```diff
@@ -0,0 +1,291 @@
+---
+title: DUNE
+section: 1
+header: User Manual
+footer: DUNE 0.0.1
+date: April 04, 2023
+---
+# NAME
```

Comment:
In general the EOS/ENF docs do not use ALL caps for titles. Instead all main words in titles use capital letters except the ones that connect the main words, e.g. 'a', 'an', 'the', 'of', etc.

Reply:
DUNE is an abbreviation so I wrote it this way. Corrected.

Reply:
> In general the EOS/ENF docs do not use ALL caps for titles.

Does DUNE fit the general case? I think all caps for the project name is unfortunate, but that's where we are.

One option for correcting this is changing the name. I hope we can address this soon. See #141

Reply:
Furthermore, after a bit of investigation, [best practices](https://man7.org/linux/man-pages/man7/man-pages.7.html) indicate the title should be all caps.

Reply:
Meets best practices now. TY